ex1/ex1.c

```diff
@@ -9,6 +9,18 @@
 int main(void)
 {
     // Your code here
+    int x = 100;
+    if (fork() == 0)
+    {
```

Comment:
When you fork, it's always good to include an error handler in case something goes wrong, so that the compiler can exit cleanly. There is an example of this in the README.